src/main/java/com/insightfullogic/honest_profiler/ports/sources/FileLogSource.java

```diff
@@ -71,8 +71,8 @@ public FileLogSource(final File file)
         this.file = file;
         try
         {
-            raf = new RandomAccessFile(file, "r");
-            channel = raf.getChannel();
+            this.raf = new RandomAccessFile(file, "r");
```

Comment:
Hm, I forgot to turn off my Save Actions for this file, which auto-format and clean up a bunch of things. I normally do this whenever editing any files outside of the ports/fx package since those are closest to the "original" code. Sorry about that. Should I reset and do it again with the minimal formatting ?

As for the incremental approach, I'm still unclear about the following : 
Say that I I create 2 branches off of some commit, and I make changes on each and issue a PR for each, i.e. the PRs are parallel.

Then afterwards if I want to use a build containing both fixes I need to merge both to my forked master. But if the PRs get accepted and merged into the main project, then the commit containing the merged changes for these 2 PRs may diverge from the one on my forked master, right ? Or does that not matter, or can git see that they are essentially the same ?